app/controllers/concerns/valid_client_concern.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module ValidClientConcern
```

Comment:
Why create another concern? This could be placed in the existing doorkeeper-related concern.

Concerns should not be over-used.

Reply:
I'll refactor this once after merging #13 🙏